README.md

```diff
@@ -17,16 +17,72 @@ standard implementation of lightweight Virtual Machines (VMs) that feel and
 perform like containers, but provide the workload isolation and security
 advantages of VMs.
 
+## License
+
+The code is licensed under an Apache 2.0 license.
+See [the license file](LICENSE) for further details.
+
+## Platform support
+
+Kata Containers currently runs on 64-bit systems supporting the following
+technologies:
+
+| Platform | Architecture | Virtualization technology |
+|-|-|-|
+| [Intel](https://www.intel.com) | `x86_64` | VT-x |
+| [AMD](https://www.amd.com) | `amd64` | SVM |
+| [ARM](https://www.arm.com) | `aarch64` ("`arm64`")| Hyp |
+| [IBM](https://www.ibm.com) Power | `ppc64le` |  |
+| [IBM](https://www.ibm.com) Z mainframes | `s390x` | SIE |
```

Comment:
@Jakob-Naucke - please could you confirm that SIE is correct here? Should this row also mention "zSeries" or "Linux on Z" or whatever the latest term is? :)

Reply:
@jodh-intel thanks for the update. Yes, the official term is IBM Z & LinuxONE, and SIE would still be correct.